qa/L0_batcher/batcher_test.py

```diff
@@ -229,12 +272,14 @@ def test_multi_batch_different_shape(self):
                 threads.append(threading.Thread(target=self.check_response,
                                                 args=(trial, 1, (3000, None)),
                                                 kwargs={'input_size': 16,
-                                                'shm_region_names': ['ip00', 'ip01', 'op00', 'op01']}))
+                                                'shm_region_names': ['ip00', 'ip01', 'op00', 'op01'],
```

Comment:
Always set `shm_region_names` even though the test may not be run with `TEST_SHARED_MEMORY`? Have to look into infer_utils.py to realize `TEST_SHARED_MEMORY` is also used in `infer_exact()`. Perhaps just set this to None like what `precreated_shm_regions` does

Reply:
Done